timed/projects/filters.py

```diff
@@ -51,6 +52,8 @@ class MyMostFrequentTaskFilter(Filter):
     """
 
     def filter(self, qs, value):
+        if value in EMPTY_VALUES:
+            return qs
```

Comment:
same as above doc string needs to come right after function. However this actually has some information and needs to be kept.